### DISCUSSION TITLE
SD Karten slot

### DISCUSSION THREAD

**Discussion by stefan190977** [tbnobody/OpenDTU#485] [Ideas]
Guten Abend,
Wäre es möglich an dem ESP-32 einen SD Karten slot anzuschliessen und den akutellen Tages verbrauch auf der Krate zu speichen ?

MFG 
und allene einen guten Rutsch ins neue Jahr

**Comment by themanfrommoon** [comment]
Sehr gut, genau das suche ich auch!

**Comment by uvcollector** [comment]
Jup, das suche ich ebenfalls.

**Comment by stefan190977** [comment]
Kurze Frage, wäre es denn möglich ?
